src/crepe/system/ScriptSystem.cpp

```diff
@@ -23,6 +24,8 @@ void ScriptSystem::update() {
 			script->init();
 			script->initialized = true;
 		}
-		script->update();
+
+		duration_t delta_time = timer.get_delta_time();
+		script->update(delta_time);
```

Comment:
Je hebt nog geen onderscheidt gemaakt tussen `fixed_update` en `frame_update`, toch?
Want (als ik mij niet vergis) wordt `update` nu aangeroepen in de `fixed_update` en dat betekend dat je `duration_t delta_time = timer.get_scaled_fixed_delta_time();` moet gebruiken.
Kun je niet direct even `fixed_update()` en `frame_update()` aan Script toevoegen, dan is het voor altijd gefixt.

Reply:
Als #78 gemerget is heb ik in ScriptSystem pas de mogelijkheid om op de fixed_update en frame_update wat uit te voeren dus dat ga ik daarna pas toevoegen aan Script.

Reply:
Dit is PR #90 geworden